executor/write_test.go

```diff
@@ -2963,3 +2964,15 @@ func (s *testSuite4) TestWriteListColumnsPartitionTable(c *C) {
 	tk.MustExec("delete from t partition (p3,p2)")
 	tk.MustQuery("select * from t order by id").Check(testkit.Rows("1 a", "2 b"))
 }
+
+func (s *testSuite7) TestIssue20724(c *C) {
```

Comment:
Please change the test suite to a serial test suite.

Reply:
OK, I have fixed it.